prql-compiler/src/codegen.rs

```diff
@@ -585,4 +585,26 @@ mod test {
         let pipeline = pl::Expr::from(pl::ExprKind::Pipeline(pl::Pipeline { exprs: vec![short] }));
         assert!(pipeline.write(opt).is_none());
     }
+
+    #[test]
+    fn test_escaped_string() {
+        use itertools::Itertools;
+
+        let escaped_string = crate::prql_to_pl(
+            r#"
+        from tracks
+        filter (name ~= "\\(I Can't Help\\) Falling")
+        "#,
+        )
+        .unwrap()
+        .into_iter()
+        .exactly_one()
+        .unwrap();
+
+        // TODO: should this require `trim`? Possibly a linebreak is being added.
```

Comment:
It would be better not to generate additional `\n` if we don't want them. Or is trim here to mitigate snapshot formatting by `cargo fmt`?

Reply:
I'm not sure, I can look. It's possible we're generating an additional line break...